i8_terminal/version.txt

```diff
@@ -1 +1 @@
-__version__ = "0.2.101"
+__version__ = "0.2.102"
```

Comment:
Please update the version and make sure the auto-complete is working with the latest bug fix, which I just merged. @sadeghhadjigi